### ISSUE TITLE
[Suggestion] Automatic Villager Reroll

### ISSUE THREAD

**Issue by deeeity** [MeteorDevelopment/meteor-client#2973]
### Describe the feature

The feature should allow the player to somehow easily reroll a villager.

I guess it could work by specifying a villager type (by clicking on the block, e.g. a lectern)
The module will then find the closest villager to that lectern and constantly send packets and read data until the player gets the desired items in the item pool of the villager. You could set the villager type and item inside the module's menu.

### Before submitting a suggestion

- [X] This feature doesn't already exist in the client. (I have checked every module and their settings on the **latest dev build**)

- [X] This wasn't already suggested. (I have searched suggestions on GitHub and read the FAQ)

**Comment by Wide-Cat** [comment]
theres an addon for it -> https://github.com/maxsupermanhd/meteor-villager-roller

**Comment by seasnale** [comment]
so? it isn't a rejected feature

**Comment by deeeity** [comment]
> theres an addon for it -> https://github.com/maxsupermanhd/meteor-villager-roller

thank you, i will try this out
however, it'd be nice to see this implemented inside meteor

**Comment by I-Got-You-Dead** [comment]
> > theres an addon for it -> https://github.com/maxsupermanhd/meteor-villager-roller
> 
> thank you, i will try this out however, it'd be nice to see this implemented inside meteor

Not really

**Comment by deeeity** [comment]
> Not really

yes really, and i have a few reasons after testing out the addon

- the addon's functionality is severly limited by the fact that it cannot directly use mixins, like an original meteor feature would
- u can not do something else while waiting for the rerolls, e.g. u can't lose focus from minecraft's window, otherwise the roller will stop (due to lack of functionality access)
- currently, it only supports enchantments, however, im sure meteor devs are able to implement the other villager types aswell
- also, the addon just breaks sometimes, idk why

**Comment by I-Got-You-Dead** [comment]
I don’t really care

**Comment by deeeity** [comment]
> I don’t really care

if you don't have anything contributing/constructive to say, it's best to just stay off the issue

**Comment by I-Got-You-Dead** [comment]
Cry

**Comment by maxsupermanhd** [comment]
> so? it isn't a rejected feature

It absolutely is and I remember someone saying that it should stay in addon form. If it is not a rejected feature then why not merge it right away?

**Comment by Ompyl** [comment]
I would also like to see this module in meteor

**Comment by arlomcwalter** [comment]
same
